barely.go

```diff
@@ -104,6 +118,26 @@ func (bar *StatusBar) Render(writer io.Writer) error {
 	return nil
 }
 
+func graphicLength(str string) int {
+	c := 0
+	for _, r := range str {
+		if unicode.IsGraphic(r) {
```

Comment:
I'm not sure that's right way to get proper display length for the status bar, because ultimately you will count two-character wide utf rune as one.

Also, it will miscount escape sequences that might present in the result string.